src/tv2-common/actions/executeAction.ts

```diff
@@ -275,6 +276,15 @@ export async function executeAction<
 			case AdlibActionType.FADE_DOWN_PERSISTED_AUDIO_LEVELS:
 				await executeActionFadeDownPersistedAudioLevels(context, settings)
 				break
+			case AdlibActionType.SELECT_ROBOT_PRESET: {
+				if (Number.isNaN(Number(triggerMode))) {
+					context.notifyUserWarning(`Calling Robot preset ignored. '${triggerMode}' is not a number`)
+					break
+				}
+				const preset = Number(triggerMode)
+				await executeActionSelectRobotPreset(context, preset)
+				break
```

Comment:
No need to convert it twice
```suggestion
				const preset = Number(triggerMode)
				if (Number.isNaN(preset)) {
					context.notifyUserWarning(`Calling Robot preset ignored. '${triggerMode}' is not a number`)
					break
				}
				await executeActionSelectRobotPreset(context, preset)
				break
```

Reply:
Updated